ignite/contrib/engines/common.py

```diff
@@ -274,11 +275,14 @@ def empty_cuda_cache(_: Engine) -> None:
     gc.collect()
 
 
-def setup_any_logging(logger, logger_module, trainer, optimizers, evaluators, log_every_iters) -> None:  # type: ignore
-    raise DeprecationWarning(
-        "ignite.contrib.engines.common.setup_any_logging is deprecated since 0.4.0. and will be remove in 0.6.0. "
-        "Please use instead: setup_tb_logging, setup_visdom_logging or setup_mlflow_logging etc."
-    )
+@deprecated(
+    "0.4.0",  # type: ignore
```

Comment:
`# type: ignore` is not correctly placed. Let's remove it.

Reply:
Actually, this is the problem I was facing with `mypy` I originally had not placed it there, but `mypy` was passing on CI only after I placed it there.
I will again try and remove it once

Reply:
I get the following error on removing the line
```bash
$  mypy --config-file mypy.ini
ignite/contrib/engines/common.py:284: error: Function is missing a type annotation for one or more arguments  [no-untyped-def]
    def setup_any_logging(logger, logger_module, trainer, optimizers, evaluators, log_every_iters) -> None:
    ^
Found 1 error in 1 file (checked 86 source files)
```
I probably think it is because the decorator takes place for the actual function, hence the line numbers are a bit weird. 
If I place the `# type : ignore` in the line mentioned in the above error i.e. L284, the mypy tests pass locally but fail on CI
That is what I was referring to [here](https://github.com/pytorch/ignite/pull/1585#issuecomment-772734896)